doc/tutorials/nvmx_virtualization.rst

```diff
@@ -0,0 +1,287 @@
+
+   - Edit ``SCHEDULER`` to ``SCHED_NOOP``
+
+   .. code-block:: xml
+      :emphasize-lines: 3, 10
+
+      <FEATURES>
+         <RELOC>y</RELOC>
+         <SCHEDULER>SCHED_NOOP</SCHEDULER>
+         <MULTIBOOT2>y</MULTIBOOT2>
+         <ENFORCE_TURNOFF_AC>y</ENFORCE_TURNOFF_AC>
+         <RDT>
+            <RDT_ENABLED>n</RDT_ENABLED>
+            <CDP_ENABLED>n</CDP_ENABLED>
+         </RDT>
+         <NVMX_ENABLED>y</NVMX_ENABLED>
+         <HYPERV_ENABLED>y</HYPERV_ENABLED>
+         <IOMMU_ENFORCE_SNP>n</IOMMU_ENFORCE_SNP>
+         <ACPI_PARSE_ENABLED>y</ACPI_PARSE_ENABLED>
+         <L1D_VMENTRY_ENABLED>n</L1D_VMENTRY_ENABLED>
+         <MCE_ON_PSC_DISABLED>n</MCE_ON_PSC_DISABLED>
+         <IVSHMEM>
+            <IVSHMEM_ENABLED>n</IVSHMEM_ENABLED>
+            <IVSHMEM_REGION/>
+         </IVSHMEM>
+      </FEATURES>
+
+Currently the Service OS's virtual UART interrupt doesn't work with LAPIC
+passthrough, which may prevent Service OS from booting. If the nVMX enabled
+guest is a Service OS, either built with SDC, Industry, or hybrid scenario,
+need to configure the scenario XML file carefully to make sure that the SOS
+boot parameter doesn't include "console=xxxx".
+
+There are two steps to accomplish this. Firstly, make sure the ``SERIAL_CONSOLE``
+setting should be empty:
+
+   .. code-block:: xml
+      :emphasize-lines: 3
+
+      <DEBUG_OPTIONS>
+        <RELEASE>n</RELEASE>
+        <SERIAL_CONSOLE></SERIAL_CONSOLE>
+        <MEM_LOGLEVEL>5</MEM_LOGLEVEL>
+        <NPK_LOGLEVEL>5</NPK_LOGLEVEL>
+        <CONSOLE_LOGLEVEL>3</CONSOLE_LOGLEVEL>
+        <LOG_DESTINATION>7</LOG_DESTINATION>
+        <LOG_BUF_SIZE>0x40000</LOG_BUF_SIZE>
+      </DEBUG_OPTIONS>
+
+Secondly, make sure the ``bootargs`` setting doesn't include any ``console=xxx``
+test, for example:
+
+   .. code-block:: xml
+      :emphasize-lines: 4,5
+
+      <board_private>
+        <rootfs>/dev/sda3</rootfs>
+        <bootargs>
+        rw rootwait consoleblank=0 no_timer_check quiet loglevel=3
+        i915.nuclear_pageflip=1 hvlog=2M@0xe00000 memmap=0x200000$0xe00000
+        </bootargs>
+      </board_private>
+
+After ACRN is build, validate this configuration from the generated code.
+For example, check this file if you build a SDC scenario:
+``acrn-hypervisor/build/hypervisor/configs/scenarios/sdc/misc_cfg.h``,
+the ``SOS_CONSOLE`` should be empty and ``SOS_BOOTARGS_DIFF`` should not
+include any ``console=xxx`` text.
+
+   .. code-block:: xml
+      :emphasize-lines: 2,6
+
+      #define SOS_ROOTFS                    "root=/dev/sda3 "
+      #define SOS_CONSOLE                   " "
+      #define SOS_COM1_BASE                 0x2F8U
+      #define SOS_COM1_IRQ                  3U
+
+      #define SOS_BOOTARGS_DIFF             "rw rootwait consoleblank=0
+            no_timer_check quiet loglevel=3 i915.nuclear_pageflip=1
+            hvlog=2M@0xe00000 memmap=0x200000$0xe00000 maxcpus=4 "
+
+- Configure system level features:
```

Comment:
We had a "first" and "second" step.  This seems like a "third" and the build step below would be a "fourth". If so, it would be better to change the "first" and "second" sentences, this step, and the build step below into a numbered list.